rasa_core/utils.py

```diff
@@ -595,27 +595,34 @@ def read_endpoints(cls, endpoint_file):
                 endpoint_file, endpoint_type="action_endpoint")
         model = read_endpoint_config(
                 endpoint_file, endpoint_type="models")
+        tracker_store = read_endpoint_config(endpoint_file, endpoint_type="tracker_store")
 
-        return cls(nlg, nlu, action, model)
+        return cls(nlg, nlu, action, model, tracker_store)
 
-    def __init__(self, nlg=None, nlu=None, action=None, model=None):
+    def __init__(self, nlg=None, nlu=None, action=None, model=None,tracker_store=None):
         self.model = model
         self.action = action
         self.nlu = nlu
         self.nlg = nlg
+        self.tracker_store = tracker_store
 
 
 class EndpointConfig(object):
     """Configuration for an external HTTP endpoint."""
 
     def __init__(self, url, params=None, headers=None, basic_auth=None,
-                 token=None, token_name="token"):
+                 token=None, token_name="token",store_type=None,db=None,user=None,password=None,timeout=None):
```

Comment:
I am not sure we should set all of them as separate arguments. Now that we start to use this for more purposes, I think it makes sense to just take all the `**kwargs` and store them on the objects property dict:
```
for k, v in kwargs.items():
  setattr(self, k, v)
```

Reply:
yeah makes sense, i had some thoughts on that as well. I will try to do that next week 👍